### DISCUSSION TITLE
Does the Gmail API library (gmail_v1) from Google APIs take time to send emails?

### DISCUSSION THREAD

**Discussion by PatricioPoncini** [googleapis/google-api-nodejs-client#3610] [Q&A]
I'm using this library in Node.js and just made four requests in quick succession, all to the same email address and with the same body. To my surprise, only the first one arrived. Why is this happening? Is this how the library works? I would like to know more, as I understand that if this endpoint were used frequently in my application, there could be emails that wouldn't be generated

This is the body of my request; as you can see, it is very simple:
```json
{
  "subject": "Example subject",
  "to": ["test@test.com"],
  "body": "<h1>Hello, World!</h1><p>This is an email body in HTML format.</p>"
}
```

**Comment by PatricioPoncini** [comment]
I think I understand. If I send the same body to the same email address, the second email won't be sent, right? Since the previous one sent is exactly the same. Can you confirm this?

**Comment by sofisl** [comment]
Since this is an API question, I'm going to transfer this to Discussions per our new issue policy. Thank you!
